mbuild/formats/charmm_writer.py

```diff
@@ -1049,6 +1049,67 @@ def _lengths_angles_to_vectors(lengths, angles, precision=6):
     return box_vectors
 
 
+def _check_fixed_bonds_angles_lists(
+    fixed_residues_bonds_angles_list,
+    fixed_bonds_angles_residues_variable_name,
+    residues,
+):
+    """Check the GOMC fixed bonds and angles lists for input errors.
+
+    Parameters
+    ----------
+    fixed_residues_bonds_angles_list : list of strings, [str, ..., str]
+        A list of the residues (i.e., molecules since GOMC currently considers a
+        a whole molecule as a residue) to have their bonds and/or angles held
+        rigid/fixed for the GOMC simulation engine.
+        In GOMC, the residues currently the molecules.
+    fixed_bonds_angles_residues_variable_name : str
```

Comment:
```suggestion
    fixed_residues_bonds_angles_variable_name : str
```
Is this variable related to the previous one (`fix_residues_bonds_angles_list`)? If so, I think their names should be ~consistent with the previous one.

Also, I am still a bit confused by the description of this variable (is the "which string" supposed to be here?), but that may just be me.

Reply:
Yeah i am a tad confused here. Is this var supposed to be a list as well? Or am I missing something here? and I agree with @daico007 , it seems like the ordering of the words in this var seems different compared to the variable defined above it. If they are related, they should use a similar naming convention.

Reply:
I see the confusion, here now.  I changed the names to be very similar to the actual commands used in the Charmm object (see below).  Please let me know if this sounds better now.

Parameters
    ----------
    gomc_fix_bonds_and_or_angles : list of strings, [str, ..., str]
        A list of the residues (i.e., molecules since GOMC currently considers a
        a whole molecule as a residue) to have their bonds and/or angles held
        rigid/fixed for the GOMC simulation engine.
        In GOMC, the residues currently the molecules.
    gomc_fix_bonds_and_or_angles_selection : str
        The name of the varible used as a string, which string, which will be
        in the error and information outputs.
    residues : list, [str, ..., str]
        Labels of unique residues in the Compound. Residues are assigned by
        checking against Compound.name.  Only supply residue names as 4 character
        strings, as the residue names are truncated to 4 characters to fit in the
        psf and pdb file.